hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/volume/OMVolumeSetOwnerRequest.java

```diff
@@ -147,18 +145,20 @@ public OMClientResponse validateAndUpdateCache(OzoneManager ozoneManager,
           .setMessage(
             "Volume '" + volume + "' owner is already '" + newOwner + "'.")
           .setSuccess(false);
-        return new OMVolumeSetOwnerResponse(omResponse.build());
+        omResponse.setSetVolumePropertyResponse(
+            SetVolumePropertyResponse.newBuilder().build());
+        omClientResponse = new OMVolumeSetOwnerResponse(omResponse.build());
+        addResponseToDoubleBuffer(transactionLogIndex, omClientResponse,
+            ozoneManagerDoubleBufferHelper);
+        return omClientResponse;
```

Comment:
Instead of return response here, set it to omClientResponse, and let finally block take care of it.

Reply:
done